Gemfile

```diff
@@ -201,7 +201,7 @@ gem 'rubyzip', require: 'zip'
 gem 'nokogiri', '>= 1.6.8'
 
 # Polyglot support
-gem 'coursemology-polyglot', '>= 0.1.0'
+gem 'coursemology-polyglot', github: 'Coursemology/polyglot'
```

Comment:
@fonglh just to check, after the evaluator is merged with worker, this should be the only place that needs to install the gem ?